runtime/Runtest/intrinsics.c

```diff
@@ -172,6 +172,12 @@ int klee_range(int begin, int end, const char* name) {
   return x;
 }
 
+void klee_prefer_cex(void *object, uintptr_t condition) { }
+
+void klee_abort() {
+  exit(1);
```

Comment:
I use `exit(1)` here instead of `abort()`, because `abort()` seemed to throw off tests, so that `not env ... %t_runner` still fails if the runner calls `abort()`.